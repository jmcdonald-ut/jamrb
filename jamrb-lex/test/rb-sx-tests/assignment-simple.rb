# I'm just a comment
v = 5
a = b = c = d = e = 4
a += 1

self.foo = nil

maybe = v == a
