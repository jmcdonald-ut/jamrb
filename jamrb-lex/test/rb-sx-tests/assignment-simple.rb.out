((Token ((1 0) on_comment "# I'm just a comment\n"))
 (Token ((2 0) on_ident "v"))
 (Token ((2 1) on_sp " "))
 (Token ((2 2) on_op "="))
 (Token ((2 3) on_sp " "))
 (Token ((2 4) on_int "5"))
 (Token ((2 5) on_nl "\n"))
 (Token ((3 0) on_ident "a"))
 (Token ((3 1) on_sp " "))
 (Token ((3 2) on_op "="))
 (Token ((3 3) on_sp " "))
 (Token ((3 4) on_ident "b"))
 (Token ((3 5) on_sp " "))
 (Token ((3 6) on_op "="))
 (Token ((3 7) on_sp " "))
 (Token ((3 8) on_ident "c"))
 (Token ((3 9) on_sp " "))
 (Token ((3 10) on_op "="))
 (Token ((3 11) on_sp " "))
 (Token ((3 12) on_ident "d"))
 (Token ((3 13) on_sp " "))
 (Token ((3 14) on_op "="))
 (Token ((3 15) on_sp " "))
 (Token ((3 16) on_ident "e"))
 (Token ((3 17) on_sp " "))
 (Token ((3 18) on_op "="))
 (Token ((3 19) on_sp " "))
 (Token ((3 20) on_int "4"))
 (Token ((3 21) on_nl "\n"))
 (Token ((4 0) on_ident "a"))
 (Token ((4 1) on_sp " "))
 (Token ((4 2) on_op "+="))
 (Token ((4 4) on_sp " "))
 (Token ((4 5) on_int "1"))
 (Token ((4 6) on_nl "\n"))
 (Token ((5 0) on_ignored_nl "\n"))
 (Token ((6 0) on_kw "self"))
 (Token ((6 4) on_period "."))
 (Token ((6 5) on_ident "foo"))
 (Token ((6 8) on_sp " "))
 (Token ((6 9) on_op "="))
 (Token ((6 10) on_sp " "))
 (Token ((6 11) on_kw "nil"))
 (Token ((6 14) on_nl "\n"))
 (Token ((7 0) on_ignored_nl "\n"))
 (Token ((8 0) on_ident "maybe"))
 (Token ((8 5) on_sp " "))
 (Token ((8 6) on_op "="))
 (Token ((8 7) on_sp " "))
 (Token ((8 8) on_ident "v"))
 (Token ((8 9) on_sp " "))
 (Token ((8 10) on_op "=="))
 (Token ((8 12) on_sp " "))
 (Token ((8 13) on_ident "a"))
 (Token ((8 14) on_nl "\n")))
